spring-cloud-task-core/src/test/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDaoTests.java

```diff
@@ -16,17 +16,19 @@
 
 package org.springframework.cloud.task.repository.dao;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
Static imports should be at the end of the list: https://github.com/spring-projects/spring-framework/wiki/Code-Style#import-statements